mobile/src/calendar/event-dialog/edit-sick-leave-event-dialog.tsx

```diff
@@ -30,15 +32,16 @@ class EditSickLeaveEventDialogImpl extends Component<EditSickLeaveEventDialogPro
     //----------------------------------------------------------------------------
     public render() {
         const selectedSickLeave = this.selectedSickLeave();
-        const isEditable = selectedSickLeave ? !selectedSickLeave.calendarEvent.isCompleted : false;
+        const isCompleted = selectedSickLeave ? !selectedSickLeave.calendarEvent.isCompleted : false;
+        const isDisabledForComplete = !(this.props.completeSickLeavePermission === true && !isCompleted);
```

Comment:
IMO, it is better to avoid inverse logic wherever it is possible. Doesn't you think that
`const isDisabledForComplete = this.props.completeSickLeavePermission === false || isCompleted;`
is much more readable?